osxcollector/osxcollector.py

```diff
@@ -1205,6 +1208,22 @@ def _collect_system_info(self):
         }
         Logger.log_dict(record)
 
+    def _collect_binary_names_in_path(self):
+        """Collect the names of executable binaries in the PATH environment"""
+        exe_files = []
+
+        def is_exe(fpath):
+            return os.path.isfile(fpath) and os.path.exists(fpath) and os.access(fpath, os.X_OK)
+
+        if PATH_ENVIRONMENT_NAME in os.environ:
+            for bin_dir in os.environ[PATH_ENVIRONMENT_NAME].split(":"):
```

Comment:
Super small nit and probably unnecessary since OS X uses colon as path separator by default, but maybe using `os.pathsep` for more readability?

Reply:
makes sense @leeren, I'll update it.